### PR TITLE
split solutions

### PR DESCRIPTION
**Pull request by colombod**
split solutions and build

### CODE REVIEW

File: eng/build.ps1 [hunk 1]
```diff
@@ -12,7 +12,7 @@ function TestUsingNPM([string] $testPath) {
 
 try {
     # invoke regular build/test script
-    . (Join-Path $PSScriptRoot "common\build.ps1") @args
+    . (Join-Path $PSScriptRoot "common\build.ps1") "-projects $PSScriptRoot\..\dotnet-interactive.sln;$PSScriptRoot\..\dotnet-try.sln"
```
Comment (brettfo) [inline eng/build.ps1]: You still have to pass through `@args`.

File: eng/build.sh [hunk 2]
```diff
@@ -36,7 +36,7 @@ function TestUsingNPM() {
 }
 
 # invoke regular build/test script
-. "$scriptroot/common/build.sh" $args
+. "$scriptroot/common/build.sh" "$scriptroot/../dotnet-interactive.sln;$scriptroot/../dotnet-try.sln"
```
Comment (brettfo) [inline eng/build.sh]: Same, pass through `$args`.

File: eng/build.ps1 (hunk 1 above)
Comment (brettfo) [reply eng/build.ps1]: Also, you _might_ have access to the variable `$RepoRoot` so you don't need `$PSScriptRoot\..`.

Comment (colombod) [reply eng/build.ps1]: then why the line below in test like `TestUsingNPM "$PSScriptRoot\..\Microsoft.DotNet.Try.Client"`

**Pull request merged**